src/hooks/privateChats.ts

```diff
@@ -248,38 +241,39 @@ export const usePrivateChatNotification = (
     interrupt: true,
   });
 
-  const privateChateMessagesSorted = useMemo(
+  const privateChatMessagesSorted = useMemo(
     () => Object.values(privateChatMessages).sort(chatSort),
     [privateChatMessages]
   );
```

Comment:
Why do we need `Object.values` here? Isn't `privateChatMessages` already an array? Would this work?

```suggestion
    () => privateChatMessages.sort(chatSort),
    [privateChatMessages]
  );
```

Reply:
If we don't use `Object.values` it gives an error `Cannot assign to read only property '0' of object '[object Array]' in typescript`, but we can use `() => [...privateChatMessages].sort(chatSort),` If you think is better,

Reply:
I think the spread syntax you suggested is more obvious in intent, that we basically want to clone the array before sorting it.

Though that raises another question for me, do we need to sort it at all? Are they already sorted correctly when they are queried from firestore?